tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DbScopeFactory.cs

```diff
@@ -245,7 +246,7 @@ static Cache()
             {
                 CommandType = typeof(TCommand);
 
-                if (TryGetIntegrationDetails(CommandType.FullName, out var integrationId, out var dbTypeName))
+                if (TryGetIntegrationDetails(Tracer.Instance.Settings.DisabledAdoNetCommandTypes, CommandType.FullName, out var integrationId, out var dbTypeName))
```

Comment:
This one is actually kind of annoying. As it's in a static cctor, we can't pass it in anywhere, which means there's _one_ `DbScopeFactoryTests` test which will use this and end up initializing the global tracer. Oh well.